Blish HUD/Strings/GameServices/OverlayService.de.resx

```diff
@@ -156,4 +156,22 @@ Guild Wars 2, Heart of Thorns und alle zugehörigen Logos und Designs sind Waren
 eingetragene Warenzeichen der NCSOFT Corporation. Alle anderen Warenzeichen sind das Eigentum
 ihrer jeweiligen Besitzer.</value>
   </data>
+  <data name="Setting_AudioDevice_Description" xml:space="preserve">
+    <value>Dies bestimmt das Audioausgabegerät welches von Blish HUD verwendet wird. Das GW2-Ausgabegerät passt sich an das Audiogerät von GW2 an.</value>
+  </data>
+  <data name="Setting_AudioDevice_DisplayName" xml:space="preserve">
+    <value>Audiogerät</value>
+  </data>
+  <data name="Setting_UseGameVolume_Description" xml:space="preserve">
+    <value>Wenn aktiviert, wird Blish HUD seine Lautstärke an die des Spiels anpassen.</value>
```

Comment:
This wording declares bhud as masculine. Should it be kept?

Reply:
This is not entirely true. In this context "seine" is the correct translation for either "his" (masculine) or "its" (neutral).

Reply:
I'm afraid I'm unfamiliar with the significance here as I only speak English.  😕 So I will have to defer on what the appropriate sentence should be.

Reply:
MrMilchmann is right. It's not clear. An alternative would be `seine` -> `die`. That has the issue that it introduces an article that references bhud's volume which only becomes clear through the second `die` schortly after. It makes the sentence a little more complex. I would probably still use `die` as `seine` diverges from how the rest of the settings descriptions are written.

Reply:
Thank you both for the feedback on this.  🙂